### ISSUE TITLE
CUxD Support

### ISSUE THREAD

**Issue by hobbyquaker** [hobbyquaker/hm2mqtt.js#1]
Does anyone want this? I don't need it.

**Comment by holgerimbery** [comment]
Yes, this would be great

**Comment by Hypnos3** [comment]
Yes, I want this do direct connect Wemos to CUxD device by using MQTT :-)

**Comment by mreschka** [comment]
I would love to have that!

**Comment by mverboom** [comment]
Would be awesome to have CUxD support, could have my heating on mqtt! :+1:

**Comment by hobbyquaker** [comment]
3.0.0 supports CUxD: `--interfaces BidCos-RF,HmIP-RF,VirtualDevices,CUxD` talks binrpc to
port 8701 and subscribes like any other interface. Untested here (no CUxD on my CCU) and
binrpc must be reachable from where hm2mqtt runs — on a CCU3 that means running hm2mqtt on the
CCU (Docker) or checking that 8701 is open. Feedback from CUxD users welcome; closing, please
open a new issue if it does not work.
